windows/src/unit-tests/standards-data/Keyman.Test.System.CanonicalLanguageCodeUtilsTest.pas

```diff
@@ -0,0 +1,88 @@
+unit Keyman.Test.System.CanonicalLanguageCodeUtilsTest;
+
+interface
+uses
+  DUnitX.TestFramework;
+
+type
+
+  [TestFixture]
+  TCanonicalLanguageCodeUtilsTest = class(TObject)
+  public
+    [Test]
+    procedure TestSomeTags;
+  end;
+
+implementation
+
+uses
+  Keyman.System.CanonicalLanguageCodeUtils;
+
+{ TCanonicalLanguageCodeUtilsTest }
+
+procedure TCanonicalLanguageCodeUtilsTest.TestSomeTags;
+begin
+  // #3485 - Gilaki (Latin) script
```

Comment:
This tests both (Arab) and (Latin) scripts.

What about also testing this?
Assert.AreNotEqual('glk-Latin-IR', TCanonicalLanguageCodeUtils.FindBestTag('glk'));

Reply:
Given `Assert.AreEqual('glk-Arab-IR', TCanonicalLanguageCodeUtils.FindBestTag('glk'));` already passed, I don't think we need to try and test the inverse as well 😁